CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ find_package(everest-cmake 0.1 REQUIRED
 )
 
 # options
-option(BUILD_TESTING "Run unit tests" OFF)
+option(FRAMEWORK_TESTING "Enable unit tests" OFF)
```

Comment:
Why are we changing the variable that CTest relies on? 

`BUILD_TESTING` is the variable used by CTest, and so I presume that we want to keep that standardized: https://cmake.org/cmake/help/latest/module/CTest.html

Reply:
The idea was to make it possible to just test specific parts of the project and not the whole, but I guess this needs to addressed in a more general way. I remove it for now…